app/views/admin/mods/index.html.erb

```diff
@@ -48,7 +48,7 @@
           <% if params[:state] == "potential" %>
             <td>
               <%= form_with model: [:admin, mod], url: admin_mod_path(mod.id), method: :patch, local: true do |f| %>
-                <%= f.submit "Make Trusted", class: "btn btn-light js-add-to-mod-channel" %>
+                <%= f.submit "Make Trusted", class: "crayons-btn js-add-to-mod-channel" %>
```

Comment:
```suggestion
                <%= f.submit "Make Trusted", class: "crayons-btn crayons-btn--secondary js-add-to-mod-channel" %>
```

I think this button shouldn't be "prmiary" one (purple) but secondary. Primary button should be "reserved" for the most important action on a page (or within certain area). 

I do agree that the button you have fixed originally could be a bit more visible and I think `crayons-btn--secondary` will be a better replacement in this case.

Reply:
![secondary_button](https://user-images.githubusercontent.com/8092120/97159928-98765f00-17a1-11eb-9dc6-d8f744685bfe.png)

This is how it looks after the above change